01-Plot-Raster.Rmd

```diff
@@ -237,44 +282,56 @@ plot(DSM_HARV,
      legend=F)
 ```
 
-Note that here we used the color palete `rainbow()` instead of `terrain.color()`.
-
->##Challenge
->Use the files in the `NEON_RemoteSensing/SJER/` to create a map of the SJER DTM
->layered on top of the SJER DTM hillshade. Then create a map of the SJER DSM layered
->on top of the SJER DSM. Experiment with various <a href="https://stat.ethz.ch/R-manual/R-devel/library/grDevices/html/palettes.html" target="_blank">R color palettes</a>
->when you make your maps.
-
-```{r challenge-plots, echo=FALSE}
+The alpha value determines how transparent the colors will be (0 being
+transparent, 1 being opaque).  Note that here we used the color palette
+`rainbow()` instead of `terrain.color()`.
+
+For information on other <a href="https://stat.ethz.ch/R-manual/R-devel/library/grDevices/html/palettes.html" target="_blank">R color palettes</a>. 
```

Comment:
I like.  And not enough to make an issue for but something for both of us to think about.  Throughout the lessons I think these short "more about" lines are most often bulleted but lets keep an eye out for those that aren't as I think a few are not!